src/test_utils.rs

```diff
@@ -66,7 +67,7 @@ pub fn create_random_block(slot: Slot, num_slices: usize) -> Vec<Slice> {
         let payload = create_random_slice_payload_valid_txs(parent);
         let header = SliceHeader {
             slot,
-            slice_index,
+            slice_index: SliceIndex::new(slice_index),
```

Comment:
we don't even need to use this here, right? we can just use `until()` here as well

Reply:
and then also use `slice_index.is_first()` above (in line 62)